input/templates/my-account-edit-address.hbs

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  {{> common/htmlhead}}
+</head>
+
+<body>
+  <div class="darkbg hidden"></div>
+  {{> common/header}}
+  <div class="address-book">
+    <div class="container">
+      <div class="my-account-title">
+        <span class="my-account-title-text icon-user">{{i18n "my-account:myAccount.title"}}</span>
+      </div>
+      <div class="row">
+        <div class="col-sm-12">
+          <div class="address-book-wrapper">
+            <div class="row">
+              <div class="col-sm-3">
+                {{> myaccount/my-account-sidebar positionTwo=true}}
+              </div>
+              <div id="my-account-desktop-content" class="col-sm-9">
+                <div class="address-book-content edit-address-book-content">
+                  <div class="row">
+                    <div class="col-sm-12">
+                      <div class="address-book-title hidden-xs">
+                        {{content.contentTitle}} > {{content.shippingAddress.edit}} {{content.shippingAddress.title}}
+                      </div>
+                    </div>
+                  </div>
+                  <div class="row">
+                    <div class="col-sm-12">
+                      {{> myaccount/edit-address-form-elements form=addressForm.billingAddress suffix="Billing"}}
+                    </div>
+                  </div>
+                  <div class="row">
+                    <div class="col-sm-12">
+                      <div class="edit-address-book-footer">
+                        <div class="btn-grey push-left">
+                          <button>{{i18n "my-account:buttons.cancelAddressBtn"}}</button>
+                        </div>
+                        <div class="add-new-address-btn push-right">
+                          <button>{{i18n "my-account:buttons.saveAddressBtn"}}</button>
```

Comment:
I think by default buttons inside forms are of type submit, but I prefer to make it explicit.